Tools/autotest/run_in_terminal_window.sh

```diff
@@ -34,7 +34,7 @@ elif [ -n "$DISPLAY" -a -n "$(which xterm)" ]; then
   if [ "$SITL_RITW_MINIMIZE" -eq 1 ]; then
       ICONIC=-iconic
   fi
-  exec xterm "$ICONIC" -xrm 'XTerm*selectToClipboard: true' -xrm 'XTerm*initialFont: 6' -n "$name" -name "$name" -T "$name" -hold -e "$@"
```

Comment:
wrong fix !;-P

Reply:
"$@" should be fine, it retains arguments as seperate. Are you setting --mavproxy-args in sim_vehicle? I thought line 660-691 in sim_vehicle would do word splitting.